contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/statsbeat_metrics/statsbeat.py

```diff
@@ -125,7 +139,7 @@ def _get_attach_metric(self, metric):
         else:
             # Not in any rp or VM metadata failed
             properties.append(LabelValue(_RP_NAMES[3]))
-            properties.append(LabelValue(""))
+            properties.append(LabelValue(_RP_NAMES[3]))
```

Comment:
This is kinda of weird, basically you are setting rp and rpid properties with "unknown" value, but this is tight to the order of the properties you appended in _get_attach_properties and the order of _RP_NAMES, are you following some standard or something?, this code looks easy to break in my perspective as it relies on the position of the string in the array and is quite hard to read as well.

Reply:
Yes in my opinion it is quite easy to break as well, but we are tied to how OpenCensus designs metrics to be used. The label keys and properties are separate arrays that match up based off of index. In OpenTelemetry, this will be much more robust as labels will most likely be dictionaries but for now we have to do things like this.